language_bindings/rust/test_files/iir_blur.cpp

```diff
@@ -0,0 +1,179 @@
+// This file defines a generator for a first order IIR low pass filter
```

Comment:
This appears to be a duplicate of `language_bindings/rust/examples/iir_blue/src/gens/iir_blur.cpp`. Is that so? Why can't there be a single file? Also, how is this file built? I see neither Make nor CMake rules.